smoke-test/src/test/java/org/opennms/smoketest/BSMAdminIT.java

```diff
@@ -61,59 +62,61 @@
     /**
      * Class to control the inputs of the "Business Service Edit"-Window
      */
-    private class BsmAdminPageEditWindow {
+    public static class BsmAdminPageEditWindow {
```

Comment:
Most of the method they need access to are *protected*, so we need to keep them in the same package. That said, I'd rather keep them grouped together here than breaking them out in *org.opennms.smoketest*.